open-craft/edx-platform
# [SE-3101] Fixes Sorting Issue with TinyMCE ADSKPlugin Variables and Squashes Changes from pooja/autodesk-s3url-plugin-changes

This PR squashes the commits that can be found here: https://github.com/open-craft/edx-platform/compare/opencraft-release/juniper.2...open-craft:pooja/autodesk-s3url-plugin-changes

In addition, it sorts the dumped JSON `additional_node_env_vars` so that it is consistent on different machines. (was contributing to the bug in SE-3401)

**JIRA tickets**: SE-3101, SE-3401

**Testing instructions**:

1. Compare changes in this PR with https://github.com/open-craft/edx-platform/compare/opencraft-release/juniper.2...open-craft:pooja/autodesk-s3url-plugin-changes
1. Make sure the only difference is the `sort_keys` in `pavelib/assets.py` for `json.dumps`

**Reviewers**
